docs/reference/sql-graphql/queries.md

```diff
@@ -97,3 +97,50 @@ const result = await res.json()
 console.log(result.data) // { countMovies : { total: { 17 } }
 ```
 
+## Pagination
+
+The Platformatic DB supports for result's pagination through input parameters: `limit` and `offset`
+
+_Example_
+
+```js
+{
+  users(limit:5, offset: 10) {
+    name
+  }
+}
+```
+
+It returns 5 users starting from position 10.
+
+### Limit
+
+By default a *limit* value (`10`) is applied to each request.
+
+Clients can override this behavior by passing a value.
+In this case the server validates the input and an error is return if exceeds the `max` accepted value (`100`).
+
+Limit's values can be customized through configuration:
+
+```json
+{
+  ...
+  "core": {
+    ...
+    "limit": {
+      "default": 50,
+      "max": 1000
+    }
+  }
+}
+```
+
+*Limit* only accepts values `>= 0`. Otherwise the default behavior is applied.
```

Comment:
I would rather prefer we throw instead.

Reply:
Code updated with this [commit](https://github.com/platformatic/platformatic/pull/220/commits/5457b71fc8211bb8f819627c4d57ddc41b683ea6)

Docs updated [here](https://github.com/platformatic/platformatic/pull/220/commits/43be26b60fb01aee3fdf1dbb0309d186b2fcab1f)